homeassistant/components/hive/climate.py

```diff
@@ -28,6 +32,15 @@
 SUPPORT_FLAGS = SUPPORT_TARGET_TEMPERATURE | SUPPORT_PRESET_MODE
 SUPPORT_HVAC = [HVAC_MODE_AUTO, HVAC_MODE_HEAT, HVAC_MODE_OFF]
 SUPPORT_PRESET = [PRESET_NONE, PRESET_BOOST]
+SERVICE_BOOST_HEATING = "boost_heating"
+ATTR_BOOST_MINUTES = "minutes"
+BOOST_HEATING_SCHEMA = vol.Schema(
+    {
+        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
+        vol.Required(ATTR_BOOST_MINUTES): cv.positive_int,
```

Comment:
We want to use our `cv.time_period` for all time interval config or service options.
```py
vol.All(
    cv.time_period,
    cv.positive_timedelta,
    lambda td: td.total_seconds() // 60
)
```

Reply:
Currently changing this now

Reply:
Complete